run.go

```diff
@@ -45,6 +45,10 @@ type RunOptions struct {
 	Entrypoint []string
 	// NetworkDisabled puts the container into its own network namespace.
 	NetworkDisabled bool
+	// ForceTerminal overrides the should-we-use-a-pty logic with a "yes".
```

Comment:
Feels like we should have one value here, which takes three option.
FORCETERM, DEFAULT (DONOTHING), FORCENOTERM

UseTerminal 

This way we avoid the case where someone sets both of your options to True.